include/GafferUI/PathColumn.h

```diff
@@ -63,6 +63,26 @@ class GAFFERUI_API PathColumn : public IECore::RefCounted, public Gaffer::Signal
 
 		IE_CORE_DECLAREMEMBERPTR( PathColumn )
 
+		enum Flags
+		{
+			ResizeInteractive = 0,
+			ResizeStretch = 1,
+			ResizeFixed = 2,
```

Comment:
Does `ResizeFixed` make sense in an API where we don't provide any methods to allow that fixed width to be specified?